libs/utils/android/workloads/viewer.py

```diff
@@ -0,0 +1,108 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2015, ARM Limited and contributors.
```

Comment:
I don't normally check this but happened to notice this time - year is wrong. Dunno if it makes any difference, probably not TBH...